protocol/0056-REWA-rewards_overview.md

```diff
@@ -462,19 +462,18 @@ Market has been trading but not yet eligible for proposer bonus.
     * Transfer 20000 USDC to `ETHUSDT | market creation | USDC` 
 * start trading in the market such that traded value for fee purposes in USDT is less than 10^6
 * During the epoch 2 let the traded value be greater than 10^6
+* Then during epoch 3 set up the following transfers with the same party as the first set:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+    * Transfer 20000 USDC to `ETHUSDT | market creation | USDC` 
```

Comment:
all of these transfers must be recurring so 1) this is not needed and 2) this is not possible because you already have an active transfer from line 462. 

It is probably worth clarifying that all of the transfers in these ACs must be recurring, using the metric market_proposer and with metric asset being either empty or specified to qualify the scope of markets to only a single asset or to all.